### PR TITLE
JAMES-2501 Add TMPFS to postgres container backing SpamAssassin

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: third-party/spamassassin/src/test/java/org/apache/james/spamassassin/SpamAssassinExtension.java [hunk 1]
```diff
@@ -51,7 +52,9 @@ public SpamAssassinExtension() {
                 .withFileFromClasspath("run.sh", "docker/spamassassin/run.sh")
                 .withFileFromClasspath("spamd.sh", "docker/spamassassin/spamd.sh")
                 .withFileFromClasspath("rule-update.sh", "docker/spamassassin/rule-update.sh")
-                .withFileFromClasspath("bayes_pg.sql", "docker/spamassassin/bayes_pg.sql"));
+                .withFileFromClasspath("bayes_pg.sql", "docker/spamassassin/bayes_pg.sql"))
+                .withCreateContainerCmdModifier(cmd -> cmd.getHostConfig()
+                    .withTmpFs(ImmutableMap.of("/var/lib/postgresql/data", "rw,noexec,nosuid,size=100m")));
```
Comment (trantienduchn) [inline third-party/spamassassin/src/test/java/org/apache/james/spamassassin/SpamAssassinExtension.java]: `postgresql`? is that legal?

Comment (chibenwa) [reply third-party/spamassassin/src/test/java/org/apache/james/spamassassin/SpamAssassinExtension.java]: Sorry? The base image is a postgres SQL image.

**Comment by mbaechler** [comment]
what is the measured gain ?

**Comment by chibenwa** [comment]
> what is the measured gain ?

No idea. That test ran on `anakin` and thus was much slower than the reference build on obiwan.

I will reschedule a build to sort this out.

test this please

(was :green_apple: )

**Comment by chibenwa** [comment]
test this please (rescheduled so that hopefully it runs on Obiwan)

**Comment by chibenwa** [comment]
Good idea but gains can be neglected. (Few seconds)

Hence I close this PR

**Pull request closed without merging**